### DISCUSSION TITLE
Ch8. Online Conformal Prediction

### DISCUSSION THREAD

**Discussion by cos18** [snuuq/conformal-prediction-seminar#8] [Discussion]
담당자: 박성우
필수 질문 마감기한 : ~ 11/13 23:59
궁금한점에 대해서 질문을 하단에 남겨주세요.

**Comment by SungheePark-Stat21** [comment]
X1, X2, X3, ..., Xt, 로부터 score function을 이용하여 C(X_t+1)을 구성하는 Conformal Prediction Flow에서 Online Comformal Prediction을 통한 추정구간의 열을 구하는 것이 실질적으로 의미가 있는 것인지 알고 싶습니다. 구간추정 시점에서 예측할 수 있는 부분은 X_(n+1)에 대한 예측구간 뿐이고 이에 대한 Conformal로 충분한 것이 아닌지 하는 생각이 듭니다.

**Comment by 2hyerim** [comment]
online conformal prediction은 split conformal에는 적용할 수 없나요? 만약 적용할 수 있다면 어떤 방식으로 매 시점의 예측집합을 만드나요?

**Comment by cys0220** [comment]
supermartingale은 교환가능성이 성립할 때 작아진다는 것과, supermartingale의 평균이 시간이 흐름에 따라 더 커지지 않는다는 게 이해가 잘 안 갑니다. supermartingale의 평균은 random variable로 bound되어있는 것이니, random variable이 아주 작은 확률로 시간이 흐름에 따라 더 큰 값을 갖고, 평균도 점점 더 커질 수 있는 것 아닌가요..? 이 두 가지에 대해 자세히 설명해주실 수 있나요?

**Comment by wlrhkd1230** [comment]
현재 online CP의 setting과, proposition 8.5. (or Thm 8.6.) 이 제가 생각하기로는 E-value 와 상당히 유사해 보입니다. 둘 사이의 관계에 대해 자세한 설명 부탁드립니다.

**Comment by ryanb01** [comment]
소단원 8.3에서 제시하는 quantile tracking 알고리즘의 update rule (식 8.8)을 보면 형태가 quantile loss에 대한 gradient descent (혹은 gradient ascent)와 닮아 보입니다. 혹시 식 8.8을 어떤 loss function의 최적화 알고리즘으로 생각할 수 있을까요? 이와 관련하여 이미 기존 문헌에 알려진 결과가 있는지, 없다면 발표자분의 생각은 어떠하신지 궁금합니다.

**Comment by Dayeon-Yoon** [comment]
제가 확률론에 대한 지식이 없어서, supermartingale 의 개념이 좀 생소한 것 같습니다. 혹시 예시를 몇 가지만 더 들어주실 수 있나요?
